src/calcurse.h

```diff
@@ -307,6 +307,7 @@ struct conf {
 };
 
 #define EMPTY_DAY_DEFAULT "--"
+#define EMPTY_EVENT_DESC_DEFAULT "(empty description)"
```

Comment:
Thanks for your contribution and sorry for the slow response. Could you please make this string localizable (i.e., _("(empty description)") instead of "(empty description)").